core/src/main/java/io/questdb/griffin/SqlCodeGenerator.java

```diff
@@ -306,6 +306,7 @@
 public class SqlCodeGenerator implements Mutable, Closeable {
     public static final int GKK_HOUR_INT = 1;
     public static final int GKK_VANILLA_INT = 0;
+    private static final String ASOF_JOIN_BINARY_SEARCH_HINT = "USE_ASOF_BINARY_SEARCH";
```

Comment:
could we move hints into owne class to not overcrowd the generator? Also for a lowercase keyed map, should constants be low-case?

Reply:
good one with lower case!